server/model/util.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	"time"
+
+	mm_model "github.com/mattermost/mattermost-server/v6/model"
+)
+
+// NewId is a globally unique identifier.  It is a [A-Z0-9] string 27
+// characters long.  It is a UUID version 4 Guid that is zbased32 encoded
+// with the padding stripped off, and a one character alpha prefix indicating the
+// type of entity or an 8 if unknown type.
+func NewID(idType string) string {
+	var prefix string
+	switch idType {
+	case "workspace":
+		prefix = "w"
+	case "board":
+		prefix = "b"
+	case "card":
+		prefix = "c"
+	case "view":
+		prefix = "v"
+	case "session":
+		prefix = "s"
+	case "user":
+		prefix = "u"
+	default:
+		prefix = "8"
+	}
+	return prefix + mm_model.NewId()
+}
+
+// GetMillis is a convenience method to get milliseconds since epoch.
+func GetMillis() int64 {
```

Comment:
These time methods are needed for next PR.  Was going to combine the millisecond timestamp PR with this one, but then saw the number of files changed and decided to keep them separate (except for the utils file).